.github/workflows/release.yml

```diff
@@ -10,4 +10,16 @@ jobs:
       contents: write
```

Comment:
I think you can remove these permissions. I believe they are being set for the default `GITHUB_TOKEN`.

Reply:
Done.